riotfile.py

```diff
@@ -2533,6 +2533,16 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ),
             ],
         ),
+        Venv(
+            name="anthropic",
+            command="pytest {cmdargs} tests/contrib/anthropic",
+            pys=select_pys(min_version="3.7", max_version="3.12"),
+            pkgs={
+                "pytest-asyncio": latest,
+                "vcrpy": latest,
+                "anthropic": [latest, "~=0.28", "~=0.26"],
```

Comment:
Testing every other minor version is excessive considering their release cadence (~1 week per minor release), let's just test `0.26` and `latest`.
```suggestion
                "anthropic": ["~=0.26", latest],
```
```suggestion
                "anthropic": [latest, "~=0.28", "~=0.26"],
```